index.js

```diff
@@ -1,3 +1,21 @@
+#!/usr/bin/env node
+
 'use strict';
+var fs = require('fs-extra'),
+  procArg = process.argv;
+
+if (require.main === module) {
+  process.stdout.write(main(fs.readJSONSync(procArg[2])))
+} else {
+  module.exports = main;
+}
 
-module.exports = require('./src/formatter');
+/**
+ * main function
+ * @param {String} input - input data stringify json
+ * @param {Object} [varNames] - names of the vars
```

Comment:
As mentioned below, I think that this needs a more descriptive name than just `varNames`. Also, could you please put examples in the README of how to use it and what the expected keys are (`reportName`, `errorCountName` and `warningCountName`)